### ISSUE TITLE
Transparent billboards/points not working as expected.

### ISSUE THREAD

**Issue by mramato** [AnalyticalGraphicsInc/cesium#2130]
From the mailing list: https://groups.google.com/d/msg/cesium-dev/_RSvkl1M9d8/fDV7_ZgBBXkJ
1. Pase this gist into Sandcastle: https://gist.github.com/rlittlefield/e7ee65ccf3f103fcc78d
2. Select "add polyline to old collection"
3. Select "add polyline to new collection"

Orient the camera so that the billboard is visual in front of both lines, you'll see a screenshot similar to the below.  I would expect both lines to behave like the line on the left.

Here's a screenshot
![image](https://cloud.githubusercontent.com/assets/1495196/4277589/4a694808-3d10-11e4-9468-7cba7156b050.png)

**Comment by bagnell** [comment]
The `billboard-transparency` branch has a hacked in fix for this. If we decide to go with two passes, I could clean it up.

**Comment by mramato** [comment]
On my laptop, I'm seeing odd borders behind label billboards, do you think it's related to this?  Or perhaps another bug?  In the below shot, the G is obviously this bug, but the odd border on the 1 seems like something different.

![image](https://cloud.githubusercontent.com/assets/1495196/7872102/4c8b4c14-0563-11e5-8f3f-2544dba78b56.png)

**Comment by mramato** [comment]
I think #3011 fixes the border issue, but I need to test it out on more machines.

**Comment by mramato** [comment]
Recently brought up again on the mailing list: https://groups.google.com/d/msg/cesium-dev/LAeE70CWecs/Y_BtC_u4DwAJ

**Comment by pjcozzi** [comment]
Could be the problem: http://www.realtimerendering.com/blog/gpus-prefer-premultiplication/

**Comment by pjcozzi** [comment]
Or part of the problem.

**Comment by mramato** [comment]
Brought up again in #3550

**Comment by haisapan** [comment]
Hi guys, I also got this issue, anybody can help or can give us a time to fix? @mramato @bagnell @

**Comment by hpinkos** [comment]
possibly related issue: https://groups.google.com/forum/?hl=en#!topic/cesium-dev/Mlqw0pR8jvg
there is a weird transparency effect when the billboard is scaled

**Comment by mramato** [comment]
Also causes #3782

**Comment by hpinkos** [comment]
Also reported here: https://groups.google.com/forum/?hl=en#!topic/cesium-dev/gipZ4pITQOs
Labels transparency is incorrect over models

``` javascript
var viewer = new Cesium.Viewer('cesiumContainer');

var entity = viewer.entities.add({
    position : Cesium.Cartesian3.fromDegrees(-123.0744619, 44.0503706),
    model : {
        uri : '../../SampleData/models/CesiumAir/Cesium_Air.glb',
        minimumPixelSize : 128,
        maximumScale : 20000
    },
    label: {
        text: 'cesium test name',
        scale:1,
        font: '15px Verdana',			
        fillColor : Cesium.Color.BLUE,
        horizontalOrigin : Cesium.HorizontalOrigin.RIGHT,
        eyeOffset: new Cesium.Cartesian3(0,0,-50)
    }
});

viewer.trackedEntity = entity;
```

**Comment by hpinkos** [comment]
Also reported here: https://groups.google.com/d/msg/cesium-dev/vEJZ8cgoz1A/wtP3-54CBQAJ

**Comment by nikakhov** [comment]
As a temporary fix, would it be possible to merge branch master into billboard-transparency?  This way billboard-transparency will be up to date.

**Comment by hpinkos** [comment]
Also seen here: https://groups.google.com/forum/?hl=en#!topic/cesium-dev/ElkPcTGhTwo

**Comment by hpinkos** [comment]
Also reported here: #4720

**Comment by billwritescode** [comment]
Some more appearances:

![point-not-clamped-marker-yes-clamped](https://cloud.githubusercontent.com/assets/10883162/20939794/485e002e-bbbe-11e6-9443-10401810f5c3.gif)
![point-yes-clamped-marker-yes-clamped](https://cloud.githubusercontent.com/assets/10883162/20939795/48a8e88c-bbbe-11e6-9024-51bdb47acce1.gif)
![billboards-putting-holes-in-entities-4](https://cloud.githubusercontent.com/assets/10883162/20939815/606049ac-bbbe-11e6-824f-50aa871ebfcc.PNG)

**Comment by pjcozzi** [comment]
This also happens for points.  From #2885:
```javascript
var viewer = new Cesium.Viewer('cesiumContainer');

for (var i=0; i<35000; i++) {
    viewer.entities.add({
        position : Cesium.Cartesian3.fromDegrees(-122.431297 +i/10, 37.773972+i/10),
        point : {
            pixelSize : 10,
            color : Cesium.Color.fromRandom({
                    alpha : 1.0,
                    minimumRed : 0.5,
                    minimumGreen : 0.5,
                    minimumBlue : 0.5
                })
        }
    });
}
```

Forum post: https://groups.google.com/forum/?hl=en#!topic/cesium-dev/5qv7Q_qudt8

**Comment by pjcozzi** [comment]
Also mentioned here: https://groups.google.com/forum/#!topic/cesium-dev/uoxSFmYmhi8

**Comment by pjcozzi** [comment]
Thanks all for the input, @haisapan, @nikakhov, @billwritescode.

@bagnell fixed this in #4886, which will be in Cesium 1.30 next Tuesday.

Let us know if you run into anything else.

**Comment by billwritescode** [comment]
This is fantastic! Thanks @pjcozzi @bagnell and @hpinkos.
